packages/mui-material/src/Slider/Slider.test.js

```diff
@@ -253,13 +287,13 @@ describe('<Slider />', () => {
       );
       fireEvent.touchMove(
         document.body,
-        createTouches([{ identifier: 1, clientX: 22, clientY: 0 }]),
+        createTouches([{ identifier: 1, clientX: 22.1, clientY: 0 }]),
       );
 
       expect(handleChange.callCount).to.equal(3);
       expect(handleChange.args[0][1]).to.deep.equal([21, 30]);
       expect(handleChange.args[1][1]).to.deep.equal([22, 30]);
-      expect(handleChange.args[2][1]).not.to.equal(handleChange.args[1][1]);
+      expect(handleChange.args[2][1]).to.deep.equal([22, 30]);
```

Comment:
This is not a deep equal comparison. Developers don't care that the reference is different, they want the right value.